packages/nodes-base/credentials/SquarespaceApi.credentials.ts

```diff
@@ -0,0 +1,36 @@
+import {
+	IAuthenticateGeneric,
+	ICredentialTestRequest,
+	ICredentialType,
+	NodePropertyTypes,
+} from 'n8n-workflow';
+
+export class SquarespaceApi implements ICredentialType {
+	name = 'squarespaceApi';
+	displayName = 'Squarespace API';
+	documentationUrl = 'squarespace';
+	properties = [
+		{
+			displayName: 'API Key',
+			name: 'apiKey',
+			type: 'string' as NodePropertyTypes,
```

Comment:
This should trigger [`cred-class-field-properties-assertion`](https://github.com/ivov/eslint-plugin-n8n-nodes-base/blob/master/docs/rules/cred-class-field-properties-assertion.md). Does it not?